src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/TagHelperServiceTestBase.cs

```diff
@@ -221,7 +221,7 @@ internal static (Queue<DocumentSnapshot>, Queue<TextDocumentIdentifier>) CreateD
             foreach (var (text, isRazor) in textArray.Zip(isRazorArray, (t, r) => (t, r)))
             {
                 var file = isRazor ? RazorFile : CSHtmlFile;
-                var document = CreateCodeDocument(text, file, tagHelpers);
+                var document = CreateCodeDocument(text!, file, tagHelpers);
```

Comment:
❗ This path is unconditionally hit for all values in `textArray`. One of the two annotations is not correct.

Reply:
Existing nullability annotations being incorrect is such a weird reason to request changes on a PR 😛

Reply:
This is the PR that first revealed that the annotations were not correct. We might not get another chance to see them so clearly. 😄

Reply:
Not saying it shouldn't be fixed, but personally I would only use Request Changes for the most dire of new badness being introduced. Red coloured icons scare me